pkg/rest/config_upload.go

```diff
@@ -92,6 +93,31 @@ func upsertDynatraceObject(client *http.Client, apiPath string, objectName strin
 			return dtEntity, err
 		}
 		dtEntity = translateSyntheticEntityResponse(entity, objectName)
+
+	} else if resp.Headers["Location"] != nil {
```

Comment:
nitpick/unrelated: for readability we should probably consider moving all these things we do conditionally into their own well named functions.

`upsertDynatraceObject` is getting horrendously long and full of conditionals..

(I mainly say this because github's hiding of lines made me originally thing this is an else if block to the `if existingObjectId != ""` condition...)

Reply:
You are right, good idea!